opendevin/events/action/browse.py

```diff
@@ -16,3 +16,15 @@ class BrowseURLAction(Action):
     @property
     def message(self) -> str:
         return f'Browsing URL: {self.url}'
+
+
+@dataclass
+class BrowseInteractiveAction(Action):
+    browser_actions: str
```

Comment:
Hi, I only see 'goto' browser_actions in this pr, is there any other type? Since if only one browser_action, then maybe `BrowseInteractiveAction` is just the same as `BrowseURLAction` ?

Reply:
We're planning on adding more actions later, but want to do it a little bit at the time as we validate that this doesn't hurt accuracy.

Reply:
For now, this is true, I only showed to the model in prompts about the usage of goto(). However, there are many other actions that can be supported: https://github.com/ServiceNow/BrowserGym/blob/main/core/src/browsergym/core/action/highlevel.py

Now this is to lay ground to this support in the future

Reply:
OK, got it.